lib/vsc/install/shared_setup.py

```diff
@@ -579,10 +643,10 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
         'version': VERSION,
         'author': [sdw, ag, jt],
         'maintainer': [sdw, ag, jt],
-        'packages': ['vsc', 'vsc.install'],
         'zip_safe': True,
         'install_requires': ['setuptools'],
         'setup_requires': ['setuptools'],
+        'excluded_ns_rpm': [], # vsc-install ships default removed ns vsc
```

Comment:
expand `ns` to `namespace`, don't leave people guessing? (at least in the comment)